titiler/stacapi/backend.py

```diff
@@ -244,7 +254,11 @@ def get_assets(
         }
         params.pop("bbox", None)
 
-        results = catalog.search(**params)
+        results = ItemSearch(
```

Comment:
we use `ItemSearch` to avoid using `Client.open` which will first fetch the `landing` page first